static/locales/fr.client.json

```
@@ -0,0 +1,761 @@
+    "TimeZone": "Fuseau horaire :",
+    "Locale": "Langue :",
+    "Currency": "Devise :",
+    "LocalCurrency": "Devise locale ({{currency}})",
+    "EngineRisk": "Engine (experimental {{span}} change at own risk):",
+    "Save": "Enregistrer",
+    "SaveAndReload": "Enregistrer et recharger"
+  },
+  "DuplicateTable": {
+    "NewName": "Nom de la nouvelle table",
+    "AdviceWithLink": "Au lieu de dupliquer les tables, il est généralement préférable de segmenter les données en utilisant des vues liées. {{link}}",
+    "CopyAllData": "Copier toutes les données en plus de la structure de la table.",
+    "WarningACL": "Seules les règles d'accès par défaut du document s’appliqueront à la copie."
+  },
+  "errorPages": {
+    "AccessDenied": "Accès refusé{{suffix}}",
+    "DeniedOrganizationDocuments": "Vous n’avez pas accès aux documents de cette organisation.",
+    "SignInWithDifferentAccount": "Vous êtes connecté en tant que {{email}}. Vous pouvez vous connecter avec un autre compte ou demander un accès à un administrateur.",
+    "SignInToAccess": "Connectez-vous pour accéder aux documents de cette organisation.",
+    "GoToMainPage": "Aller à la page principale",
+    "SignIn": "Connexion",
+    "AddAcount": "Ajouter un compte",
+    "SignedOut": "Déconnexion{{suffix}}",
+    "SignedOutNow": "Vous êtes maintenant déconnecté.",
+    "GenericError": "Erreur{{suffix}}",
+    "SignedInAgain": "Reconnectez-vous",
+    "PageNotFound": "Page non trouvée{{suffix}}",
+    "NotFoundMainText": "La page demandée n’a pas pu être trouvée.{{separator}}Veuillez vérifier l’URL et réessayer.",
+    "ContactSupport": "Contacter le support",
+    "SomethingWentWrong": "Une erreur s’est produite",
+    "ErrorHappened_message": "Une erreur s’est produite : {{message}}",
+    "ErrorHappened_unknown": "Une erreur inconnue s’est produite."
+  },
+  "FieldConfig": {
+    "ColumnLabel": "LABEL ET ID DE LA COLONNE",
+    "ColumnOptionsLimited": "Les options des colonnes sont limitées dans les tableaux récapitulatifs.",
+    "ColumnType_formula_one": "Colonne formule",
+    "ColumnType_data_one": "Colonne de données",
+    "ColumnType_empty_one": "Colonne vide",
+    "ColumnType_formula_other": "Colonnes formule",
+    "ColumnType_data_other": "Colonnes de données",
+    "ColumnType_empty_other": "Colonnes vides",
+    "ColumnType_mixed_other": "Comportement mixte",
+    "ConvertColumn_formula": "Effacer et transformer en formule",
+    "ConvertColumn_data": "Convertir la colonne en données",
+    "ConvertColumn_triggerformula": "Convert to trigger formula",
+    "ClearAndReset": "Effacer et réinitialiser",
+    "EnterFormula": "Saisir la formule",
+    "ColumnBehavior": "NATURE DE COLONNE",
+    "SetFormula": "Définir la formule",
+    "SetTriggerFormula": "Définir une formule d’initialisation",
+    "MakeIntoDataColumn": "Transformer en colonne de données",
+    "TriggerFormula": "TRIGGER FORMULA"
+  },
+  "ExampleInfo": {
```

```suggestion
  "ExampleCard": {
```

This is tied to the other comment about moving translation to ExampleCard.ts.